<launch>

  <arg name="HsFsmBridgeName"    default="VoiceCtlRobTask"/>
  <arg name="TaskServer"    default="VoiceCtlRob_TaskServerCmd"/>
  <arg name="TaskName"  default="VoiceCtlRob"/>
  <arg name="taskResTopName"  default="VoiceCtlRobResState"/>

  <node pkg="HsFsmBridge" type="HsFsmBridge" name="${HsFsmBridgeName}" output="screen">
     <param name="hsc3_robotIP" type="str" value="$(arg hsc3_robotIP)"/>
     <param name="hsc3_robotPort" type="int" value="$(arg hsc3_robotPort)"/>
  </node>

</launch>

